### PR TITLE
adds springer nature brand heading settings and associated utility classes

### PR DESCRIPTION
**Pull request by sky-jack**

### CODE REVIEW

**Comment by alexkilgour** [comment]
grab me about this Jack!

File: toolkits/springernature/packages/springer-nature-context/scss/30-mixins/typography.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
+@mixin u-font-smoothing() {
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+}
```
Comment (alexkilgour) [inline toolkits/springernature/packages/springer-nature-context/scss/30-mixins/typography.scss:4]: shouldn't need this as we get something very similar from `default`
https://github.com/springernature/frontend-toolkits/blob/master/context/brand-context/default/scss/30-mixins/typography.scss#L13

File: context/brand-context/springernature/scss/40-base/typography.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	@include u-h3();
+}
+
+h4 {
+	@include u-h4();
+}
+
+h5 {
+	@include u-h4();
+}
```
Comment (alexkilgour) [inline context/brand-context/springernature/scss/40-base/typography.scss:23]: should this be `u-h5()`? the mixin exists

File: context/brand-context/springernature/scss/40-base/typography.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+h6 {
+	@include u-h4();
+}
```
Comment (alexkilgour) [inline context/brand-context/springernature/scss/40-base/typography.scss:27]: there is no `u-h6()` so not sure if this is right?

File: context/brand-context/springernature/scss/40-base/typography.scss (hunk 2 above)
Comment (sky-jack) [reply context/brand-context/springernature/scss/40-base/typography.scss:23]: It should!

File: context/brand-context/springernature/scss/enhanced.scss [hunk 4]
```diff
@@ -11,4 +11,8 @@
 
 // 40-base
 @import '40-base/layout';
-@import '40-base/links';
\ No newline at end of file
+@import '40-base/links';
+@import '40-base/typography';
+
+// 60-utilities
+@import ' 60-utilities/typography';
```
Comment (alexkilgour) [inline context/brand-context/springernature/scss/enhanced.scss:18]: don't include the utilities level in the context output - these get included by the product using the context and appear after everything else (so they can override component styles)

Comment (sky-jack) [reply context/brand-context/springernature/scss/enhanced.scss]: gotcha

File: context/brand-context/HISTORY.md [hunk 5]
```diff
@@ -1,14 +1,17 @@
...
+
 ## 1.0.1 (2020-05-06)
 	* BUG: 30-mixins/links.scss missing from abstracts.scss in default
 
 ## 1.0.0 (2020-05-04)
     * Combines configurations from previous brand specific context packages
         - global-context
         - nature-context
-        - springer-context
+        - springer-contex
```
Comment (alexkilgour) [inline context/brand-context/HISTORY.md:13]: ??

File: context/brand-context/package.json [hunk 6]
```diff
@@ -1,8 +1,8 @@
 {
   "name": "@springernature/brand-context",
-  "version": "1.0.1",
+  "version": "1.1.0",
   "license": "MIT",
   "description": "Bootstrapping for all components and products",
   "keywords": [],
   "author": "Springer Nature"
-}
+}g
```
Comment (alexkilgour) [inline context/brand-context/package.json:8]: remove `g`

**Comment by alexkilgour** [comment]
Can we first remove the changes from the old `springer-nature-context` as I am going to delete the code for those packages.

**Comment by sky-jack** [comment]
Just done so

**Pull request merged**